File: python/lc26.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        k = 0
        nxt = 1
        while nxt < len(nums):
            if nums[k] == nums[nxt]:
                nxt += 1
            else:
                nums[k+1] = nums[nxt]
                k += 1
                nxt += 1
        return k+1

File: cs/lc26.cs
public class Solution {
    public int RemoveDuplicates(int[] nums) {
        if( nums.Length <=1)
            return nums.Length;
        int idx = 1;
        int i = 1;
        while(i < nums.Length)
        {
            if(nums[i] == nums[i-1])
            {
                i++;
            }
            else
            {
                nums[idx] = nums[i];
                idx++;
                i++;
            }
        }
        return idx;
    }
}
